R/utils.R

```diff
@@ -147,15 +136,11 @@ call_fun_dots <- function(fun, str_args) {
 #'
 #' @param var_names (`character`) Name of variable to extract labels from.
 #' @param dataset (`dataset`) Name of analysis dataset.
-#' @param prefix (`character`) String to paste to the beginning of the
-#'   variable name with label.
-#' @param suffix (`character`) String to paste to the end of the variable
-#'   name with label.
-#' @param wrap_width (`numeric`) Number of characters to wrap original
-#'   label to. Defaults to 80.
+#' @param prefix (`character`) String to paste to the beginning of the variable name with label.
+#' @param suffix (`character`) String to paste to the end of the variable name with label.
+#' @param wrap_width (`numeric`) Number of characters to wrap original label to. Defaults to 80.
 #'
 #' @return (`character`) String with variable name and label.
-#' @keywords internal
```

Comment:
This was removed, and function is visible in the index. Should this unexported function have an entry in the index?
<img width="455" alt="image" src="https://github.com/insightsengineering/teal.modules.general/assets/133694481/05633e72-b34a-492e-b801-aa874381bf34">

Reply:
My mistake.